### DISCUSSION TITLE
Port information previously returned in V2's "ShopAbout" endpoint to the V3 "Shop" endpoint

### DISCUSSION THREAD

**Discussion by rajivraman** [etsy/open-api#1020] [Ideas]
There is currently no way to get fields like "related-links" that were part of V2's "ShopAbout" endpoint (https://www.etsy.com/developers/documentation/reference/shopabout). I'd appreciate if this information was ported over. See related Q&A thread: https://github.com/etsy/open-api/discussions/1015

**Comment by Gareth064** [comment]
Hi @rajivraman I'm confused. Isn't this a duplicate of the thread you just linked #1015 ?

As per my advice in that thread, you need to email Etsy devs about this. developer@etsy.com

They don't take requests, or do bug tracking on this discussion board.

**Reply by rajivraman** [reply]
Hi @Gareth064 , I did reach out to support and they responded:

> Hi Raj,
>  
> We appreciate you reaching out to us.
>  
> If you wish to have your request documented so you can track it, please start a discussion on [Github Discussions](https://github.com/etsy/open-api/discussions). But rest assured that I have brought up your desire for this information with our internal engineering team. They have noted it and will use this feedback as they evaluate possible future updates.
>  
> Please let me know if you need help with anything else.
> 
> Kind regards,
> 
> V

I was a bit confused since I had linked to https://github.com/etsy/open-api/discussions/1015 in my original email.

**Reply by Gareth064** [reply]
Hmmm. I have never heard that advice from them before.

If they want people to start logging requests/ideas officially on GitHub then I think we need a better structured discussion template. Something that helps capture relevant information that is easily understood etc.

It would be crap if an official request or idea was a one-liner that no-one understands. (Not talking about yours 🤣 but you get what I mean)

**Reply by rajivraman** [reply]
Agreed. Another person responded to my email stating that the eng team has taken note of it. So I'll close this discussion.
